recipes/embree3/all/conanfile.py

```diff
@@ -0,0 +1,153 @@
+
+import os
+from conans import CMake, tools
+from conans.tools import Version
+from conans.model.conan_file import ConanFile
+from conans.errors import ConanInvalidConfiguration
+import glob
+
+
+class Embree(ConanFile):
+    name = "embree3"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "embree", "raytracing", "rendering")
+    description = "Intel's collection of high-performance ray tracing kernels."
+    generators = "cmake", "cmake_find_package"
+    homepage = "https://embree.github.io/"
+    _source_folder = "source_subfolder"
+    _build_folder = "build_subfolder"
+    exports_sources = "CMakeLists.txt"
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "geometry_curve": [True, False],
+        "geometry_grid": [True, False],
+        "geometry_instance": [True, False],
+        "geometry_quad": [True, False],
+        "geometry_subdivision": [True, False],
+        "geometry_triangle": [True, False],
+        "geometry_user": [True, False],
+        "ray_packets": [True, False],
+        "ray_masking": [True, False],
+        "backface_culling": [True, False],
+        "ignore_invalid_rays": [True, False],
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "geometry_curve": True,
+        "geometry_grid": True,
+        "geometry_instance": True,
+        "geometry_quad": True,
+        "geometry_subdivision": True,
+        "geometry_triangle": True,
+        "geometry_user": True,
+        "ray_packets": True,
+        "ray_masking": False,
+        "backface_culling": False,
+        "ignore_invalid_rays": False,
+    }
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+    
+    def configure(self):
+        if self.settings.compiler == "apple-clang" and self.settings.compiler.version == "9.1":
+            self.options.shared = True
```

Comment:
raise `ConanInvalidConfiguration` instead?

Reply:
Just realize I don't need it.  I already checked it below.

Reply:
removed